.github/workflows/checks.yml

```diff
@@ -0,0 +1,63 @@
+name: CI
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/checks.yaml'
+      - '**.go'

```

Comment:
Why limit to those paths? A change in the YAML or JSON files would force a new generation which might not get caught by this filter.

Reply:
True, but the changed/added generated code should still be `.go`
I'll remove it anyway, I think this was from the Terraform build example.